client/src/app/app-routing.module.ts

```diff
@@ -1,7 +1,33 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { ActiveDayComponent } from "./components/active-day/active-day.component";
+import {HomeComponent} from "./components/home/home.component";
+import {SigninFormComponent} from "./user/signin-form/signin-form.component";
+import {SignupFormComponent} from "./user/signup-form/signup-form.component";
+import {TeachersComponent} from "./components/teachers/teachers.component";
 
-const routes: Routes = [];
+const routes: Routes = [
+  {
+    path:'',
+    component: HomeComponent
+  },
+  {
+    path:'SignIn',
+    component: SigninFormComponent
+  },
+  {
+    path:'SignUp',
+    component: SignupFormComponent
+  },
+  {
+    path:'Teacher',
+    component: TeachersComponent
+  },
+  {
+    path:'ActiveDay',
+    component: ActiveDayComponent
```

Comment:
What is it?

Reply:
It's path and component wich will be use when we route to ActiveDay page

Reply:
What is ActiveDay?

Reply:
ActiveDay it's a new name for Event, we changed it on back end side. Entity had name Event, but you sayd that it bad name and we changed it to ActiveDay